src/runtime_src/core/edge/user/aie/aie.cpp

```diff
@@ -32,6 +32,8 @@
 
 namespace zynqaie {
 
+std::mutex am_lock;
```

Comment:
It looks like there is a comment missing here, right?

Reply:
Added comments. Thanks